iohk/NixOps.hs

```diff
@@ -445,8 +445,11 @@ instance ToJSON NodeType
 topoNodes :: SimpleTopo -> [NodeName]
 topoNodes (SimpleTopo cmap) = Map.keys cmap
 
+topoCores :: SimpleTopo -> [NodeName]
+topoCores = (fst <$>) . filter ((== NodeCore) . snType . snd) . Map.toList . fromSimpleTopo
+
 topoNCores :: SimpleTopo -> Int
-topoNCores (SimpleTopo cmap) = Map.size $ flip Map.filter cmap ((== NodeCore) . snType)
+topoNCores = length . topoCores
```

Comment:
This function seems redundant now

Reply:
You seem to refer to the fact that `topoNCores` should now be inlined (in two locations, as of now)?

I find it arguable, but OK.  Done.